runtime/src/lib.rs

```diff
@@ -210,8 +212,7 @@ impl pallet_aura::Config for Runtime {
 }
 
 impl pallet_grandpa::Config for Runtime {
-    type Event = Event;
-    type Call = Call;
+    type RuntimeEvent = RuntimeEvent;
```

Comment:
Should there not be a `type RuntimeCall` ?

Reply:
yes but we are removing a call the pallets runtime config and not replacing it?  This doesn't feel correct.